### PR TITLE
Polls 11-01

### PR DESCRIPTION
**Pull request by Hipogeo**
@prose11

### CODE REVIEW

File: governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021
+summary: Signal your support or opposition to prioritising onboarding $asset_code ($asset_name).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md:3]: Need to add asset code and asset name here @Hipogeo

File: governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 1 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset OHM (Olympus DAO) - November 1, 2021 should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md:27]: Don't think we need the date in this line

**Review by mkrorbrkr — CHANGES_REQUESTED** [review]
@Hipogeo @prose11 I've reviewed the 2 greenlight polls, just 2 small changes needed

File: governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md (hunk 2 above)
Comment (Hipogeo) [reply governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md:27]: Hey Mkr, im out right now. Ill males the changes in a couple or hours. If thats too late feel free to do them yourself

Comment (prose11) [reply governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md:27]: I’ll be doing the final review submission in ~8 hours so no rush! Just let me know if you don’t have a chance to update

Comment (mkrorbrkr) [reply governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md:27]: Don't think I have write access to your repo

**Pull request merged**